devops/tmpnightly.yml

```diff
@@ -0,0 +1,80 @@
+trigger: none
```

Comment:
why is it called tmpnightly?

Reply:
devops pipeline runs kept complaining when I tested nightly.yml
Since the test worked, I changed this to nightly.yml and I'll update the nightly pipelines etc.